pvm/ts/src/engine/texasHoldem.ts

```
@@ -518,22 +510,22 @@ class TexasHoldemGame implements IPoker {
         switch (action) {
             // todo: ante
             case PlayerActionType.SMALL_BLIND:
-                const smallBlind = new SmallBlindAction(this, this._update).execute(player, this._smallBlind);
+                const smallBlind = new SmallBlindAction(this, this._update).execute(player, this._gameOptions.smallBlind);
                 break;
             case PlayerActionType.BIG_BLIND:
-                const bigBlind = new BigBlindAction(this, this._update).execute(player, this._bigBlind);
+                const bigBlind = new BigBlindAction(this, this._update).execute(player, this._gameOptions.bigBlind);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)